server/controllers/finance/reports/configurable_analysis_report/report.handlebars

```diff
@@ -0,0 +1,154 @@
+<!doctype html>
+<html>
+  {{> head title="{{translate 'REPORT.CONFIGURABLE_ANALYSIS_REPORT.TITLE' }}" }}
+  <body>
+    <div style="margin-right: 5%; margin-left: 5%;">
+      {{> header}}
+
+      <!-- body -->
+      <div class="row">
+        <div class="col-xs-12">
+          <!-- page title  -->
+          <h3 class="text-center text-bold text-uppercase">{{translate 'REPORT.CONFIGURABLE_ANALYSIS_REPORT.TITLE'}}</h3>
+          <h4 class="text-center text-uppercase">
+            {{translate period.period_start_label}} {{period.start_year}} - {{translate period.period_end_label}} {{period.end_year}}
+          </h4>
+
+          {{#each this.type}}
+            <table style="width: 100%; font-size: 12px;" class="table table-striped table-condensed table-report table-bordered">
+              <tr>
+                <td colspan="8" class="text-center text-uppercase text-bold" style="font-size: 13.5px; background:#ededed" > {{translate label}} </td>
+              </tr>
+              {{#if is_income_expense }}
+                <tr class="text-center text-uppercase text-bold" style="font-size: 13.5px;">
+                  <td width="61%" colspan="5" class="text-right text-uppercase text-bold"></td>
+                  <td class="text-center" width="13%"> {{translate 'FORM.LABELS.DEBIT'}} </td>
+                  <td class="text-center" width="13%"> {{translate 'FORM.LABELS.CREDIT'}} </td>
+                  <td class="text-center" width="13%"> {{translate balance_type }} </td>
+                </tr>
+
+                <tr style="font-size: 13.5px;">
+                  <td width="61%" colspan="5" class="text-right text-uppercase text-bold"></td>
+                  <td width="13%" class="text-right text-uppercase text-bold"> {{debcred totalDebit ../currencyId}} </td>
+                  <td width="13%" class="text-right text-uppercase text-bold"> {{debcred totalCredit ../currencyId}} </td>
+                  <td width="13%" class="text-right text-uppercase text-bold"> {{debcred totalBalance ../currencyId}} </td>
+                </tr>  
+        
+                {{#each report}}
+                  {{#if ../../options.display_details_types}}
+                    <tr class="text-uppercase text-bold">
+                      <td width="2%" rowspan="2" width="2%"></td>
+                      <td width="59%" rowspan="2" colspan="4" class="text-uppercase text-bold"> {{translate label}} </td>
+                      <td class="text-center" width="13%"> {{translate 'FORM.LABELS.DEBIT'}} </td>
+                      <td class="text-center" width="13%"> {{translate 'FORM.LABELS.CREDIT'}} </td>
+                      <td class="text-center" width="13%"> {{translate displayLabel}} </td>  
```

Comment:
I tried to remove it but it may confuse the headers of each subcategory, and it also avoids that we can be forced to search the first page to know the title of the headers